packages/cadl-typescript/test/integration/generated/models/propertyTypes/src/models.ts

```diff
@@ -94,3 +94,9 @@ export interface DictionaryStringProperty {
   /** Property */
   property: Record<string, string>;
 }
+
+/** Model with a property never. (This property should not be included). */
+export interface NeverProperty {
+  /** Property */
+  property: never;
+}
```

Comment:
I think the proposal is to remove this never type property ?

Reply:
Will fix it in this issue: https://github.com/Azure/autorest.typescript/issues/1648